code/ZAS/ConnectionGroup.dm

```diff
@@ -215,7 +215,7 @@ Class Procs:
 	. = ..()
 
 /connection_edge/unsimulated/erase()
-	A.edges.Remove(src)
+	LAZYADD(A.edges, src)
```

Comment:
`.Remove` => `LAZYADD()` ???

_Doing things at 2AM is the best answer, obviously._